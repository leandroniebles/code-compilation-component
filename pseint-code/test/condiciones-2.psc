proceso condiciones
  s1<-"hola"
  s2<-"chau"
  n1<-10
  n2<-5.5
  n3<-1/2
  
  escribir "V ",n1>n2 y s1>s2
  escribir "V ",n1>n2 o s1<s2
  escribir "V ",n1<n2 o s1>s2
  escribir "F ",n1<n2 o s1<s2
  escribir "F ",n1>n2 y s1<s2
  escribir "F ",n1<n2 y s1>s2
  escribir "F ",n1<n2 = s1>s2
  escribir "V ",n1>n2 = s1>s2
  escribir "F ",n1<n2 y s2<s1 o n1>n2 y s2>s1
  escribir "F ",(n1>n2 y s2>s1 o n1<n2) y s2<s1
  escribir "F ",n1>n2 y (s2>s1 o n1<n2 y s2<s1)
  escribir "V ",(n1>n2 y s2>s1 o n1<n2) o s2<s1
  escribir "V ",n1>n2 o (s2>s1 o n1<n2 y s2<s1)
  escribir "F ",(n1>n2 y s2>s1) o (n1<n2 y s2<s1)
  escribir "F ",n1>n2 y (s2>s1 o n1<n2) y s2<s1
finproceso
