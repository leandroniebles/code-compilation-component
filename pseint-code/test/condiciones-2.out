V VERDADERO
V VERDADERO
V VERDADERO
F FALSO
F FALSO
F FALSO
F FALSO
V VERDADERO
F FALSO
F FALSO
F FALSO
V VERDADERO
V VERDADERO
F FALSO
F FALSO
